diff --git a/src/Settings.php b/src/Settings.php
index a3a4a17..bbbd810 100644
--- a/src/Settings.php
+++ b/src/Settings.php
@@ -136,6 +136,7 @@ class Settings
             // phpcs:ignore PHPCompatibility.Constants.NewConstants.php_binaryFound
             $settings->phpExecutable = PHP_BINARY;
         }
+        var_dump($arguments);
 
         foreach ($arguments as $argument) {
             if ($argument[0] !== '-') {
